apps/web/pages/_document.tsx

```diff
@@ -33,7 +33,7 @@ class MyDocument extends Document<Props> {
               async
               defer
               data-domain={domain}
-              src="https://plausible.io/js/plausible.js"
+              src="https://plausible.io/js/plausible3.js"
```

Comment:
expected...?

Reply:
yes, just to trigger web container rebuild.